src/main/java/org/zalando/jackson/datatype/money/MoneyModule.java

```diff
@@ -64,22 +66,71 @@ public void setupModule(final SetupContext context) {
         deserializers.addDeserializer(NumberValue.class, new DecimalNumberValueDeserializer());
         deserializers.addDeserializer(MonetaryAmount.class, new MonetaryAmountDeserializer<>(amountFactory, names));
 
+        // for reading into concrete implementation types
         deserializers.addDeserializer(Money.class, new MonetaryAmountDeserializer<>(new MoneyFactory(), names));
         deserializers.addDeserializer(FastMoney.class, new MonetaryAmountDeserializer<>(new FastMoneyFactory(), names));
-        deserializers.addDeserializer(RoundedMoney.class, new MonetaryAmountDeserializer<>(new RoundedMoneyFactory(), names));
+        deserializers.addDeserializer(RoundedMoney.class, new MonetaryAmountDeserializer<>(new RoundedMoneyFactory(Monetary.getDefaultRounding()), names));
 
         context.addDeserializers(deserializers);
     }
 
-    public MoneyModule withNumberValueSerializer(final JsonSerializer<NumberValue> numberValueSerializer) {
+    public MoneyModule withDecimalNumbers() {
```

Comment:
The main question is: do you want this factoy methods be a part of Module interface or it would be better to introduce dedicated factories like `NumberValueSerializers`, `MoneyFormatFactories`, etc